chessence-frontend/src/pages/Login/LoginPage.js

```diff
@@ -2,22 +2,55 @@ import { useState, useRef, useEffect } from "react";
 import { useMsal } from "@azure/msal-react";
 import LoginBox from "./LoginBox";
 import { loginRequest } from '../../authConfig';
+import { isAuthenticated, logoutUser } from "../../api/session";
 
 export default function LoginPage() {
-  const { instance, accounts, inProgress } = useMsal();
-  const redir = async () => {
-    try {
-      const loginResponse = await instance.loginRedirect(loginRequest);
-  } catch (err) {
-      // handle error
-      console.log(err)
-  }
-  }
-  return (
-    <div id="loginPage">
-        <h1>chessence</h1>
-        <LoginBox/>
-        <img src="/msft-login.svg" onClick={redir} />
-    </div>
-  );
+    const { instance, accounts, inProgress } = useMsal();
+    const redir = async () => {
+        try {
+            const loginResponse = await instance.loginRedirect(loginRequest);
+        } catch (err) {
+            // handle error
+            console.log(err);
+        }
+    };
+
+    // check if user is logged in
+    const [loggedIn, setLoggedIn] = useState(false);
+    useEffect(() => {
+        const fetchData = async () => {
+            const res = await isAuthenticated();
+            setLoggedIn(res);
+        };
+        fetchData().catch(console.error);
+    });
+    const logout = async () => {
+        const res = await logoutUser();
+        console.log(res);
+        setLoggedIn(false);
+    };
+
+    function LoginContent() {
+        if (loggedIn) {
```

Comment:
Is the plan to have this redirect to the home page once it is implemented?

Reply:
Yeah I think eventually it would be cool to have a `<Auth/>` component that displays different pages depending on if the user is logged in or not.